### DISCUSSION TITLE
Opinionated proposal to make front page more enticing

### DISCUSSION THREAD

**Discussion by GuillaumeDesforges** [DeterminateSystems/zero-to-nix#156] [Ideas]
In my opinion, the front page is not very enticing absolute beginners to Nix.

For instance, in my experience, it is quite uncommon for people to pay attention to

> Declarative, reproducible development environments"

However

> No more "works on my machine."

Catches quickly the attention.

See https://github.com/DeterminateSystems/zero-to-nix/blob/afcea80cec7b3a55c0330091928d3afdfb7b4703/src/site.ts#L56-L58

A suggestion would be to change place these texts.

For example:

```diff
-      title: "Declarative, reproducible development environments",
-      description:
-        'No more "works on my machine." Create environments that work seamlessly and are easily sharable across platforms.',
+      title: 'No more "works on my machine"',
+      description:
+        "Create environments that work seamlessly and are easily sharable across platforms: Nix is declarative and reproducible.",
```

Basically, go from the problems, not the solution.

**Comment by lucperkins** [comment]
@GuillaumeDesforges An intriguing suggestion! I'll think on this a bit because my first "gut" impression of your approach is good 😄
